### PR TITLE
feat(react-kit): ability to define custom ui for popup

### PR DESCRIPTION
**Pull request by dramoturg**
i need have draggable popup. for this ability i need div.container wrap with other component. 
like as Draggable(react-draggable), Rnd (react-rnd). after discussed with @raveclassic  it was decided split popup on two parts. ui and portal + root close

### CODE REVIEW

File: packages/react-kit/src/components/Popup/popup.ui.component.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+	shouldCloseOnClickAway?: boolean;
+};
+
+export type TRawPopupUiFullProps = TRawPopupUiProps & {
+	innerRef?: (instance: ReactRef) => void;
+	onBackdropClick: MouseEventHandler<HTMLElement>;
+};
+
+@PURE
+export class PopupUi extends Component<TRawPopupUiFullProps> {
```
Comment (raveclassic) [inline packages/react-kit/src/components/Popup/popup.ui.component.tsx]: nice, let's rename all `Ui`s to `UI` :)

Comment (dramoturg) [reply packages/react-kit/src/components/Popup/popup.ui.component.tsx]: done

File: packages/react-kit/src/components/Popup/popup.ui.component.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
```
Comment (sutarmin) [inline packages/react-kit/src/components/Popup/popup.ui.component.tsx]: let's rename this file to `popup-ui.component.tsx`

**Comment by sutarmin** [comment]
Hm, we changed existing component API here and we should have noticed it, but we didn't. Commit message does not reflect this fact (there is no BREAKING CHANGE part in it). We definitely should publish next version as minor, not patch, but if we do that as it is, we would lose important info in CHANGELOG. I'm not sure, how should we deal with this, need help. cc @raveclassic @mankdev

**Comment by raveclassic** [comment]
rebase, force-push and take care henceforth

**Pull request merged**